### ISSUE TITLE
Offer weights for NBEATS trained on M4 and M3 dataset for differen forecast horizons

### ISSUE THREAD

**Issue by sotte** [unit8co/darts#903]
[The NBEATs paper describes](https://arxiv.org/abs/1905.10437), that they train the model over multiple big datasets (M4, M3, TOURISM) for different forecast horizons (Yearly, Monthly, Quarterly, Weekly, Daily, Hourly). I think it would be great if darts offered loading weights from a retrained model (for a given granularity and horizon).

**Comment by hrzn** [comment]
Absolutely! This is something we've had in mind for quite some time (we're also actually about to release a blog post on this topic - see notebook [here](https://github.com/unit8co/darts/blob/master/examples/14-transfer-learning.ipynb)).
I don't think we have an issue yet to track this, so I'm moving this one to our backlog.
